src/store/index.js

```diff
@@ -4,12 +4,14 @@ import migrations from './migrations';
 import modules from './modules';
 import configPersistPlugin from './plugins/persisted-config';
 
-
 Vue.use(Vuex);
 
 const store = new Vuex.Store({
   modules,
-  plugins: [configPersistPlugin('config')],
+  // TODO: this plugin just copy the config state,
+  // if the backup will fail, we will have two configs- filesystem and store
+  // Maybe we need to backup in the action
```

Comment:
See https://github.com/brafdlog/budget-tracking/issues/51#issuecomment-658835043
I'm keeping this comment until we discuss and decide if we want to change this.